src/js/Alert.js

```diff
@@ -6,15 +6,11 @@ import Image from './Templates/Shared/Image'
 import StaticIcon from './Templates/Shared/StaticIcon'
 import store from './store.js'
 import UIController from './Controllers/UIController'
-import TTSController from './Controllers/TTSController'
-import { connect } from 'react-redux'
 
-import { updateResetPeriod, resetTimeout } from './actions';
 
-
-export const DEFAULT_RESET_TIMEOUT = 10000
+export const DEFAULT_RESET_TIMEOUT = 10000;
```

Comment:
@LiliiaShlikhtLuxoft Why does the value export?

Reply:
@ValeriiMalkov unused const - removed